src/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -43,22 +43,12 @@ public void TestChangesInConsoleEncoding()
 
             try
```

Comment:
The current console tests have coverage for a simple ASCII strings only: https://github.com/dotnet/corefx/blob/bffef76f6af208e2042a2f27bc081ee908bb390b/src/System.Console/tests/SyncTextReader.cs#L15

This change has a number of interesting paths that handle non-ASCII strings.